CHANGELOG.md

```diff
@@ -1,6 +1,15 @@
 # Change Log
 
-All notable changes to Proksee will be documented in this file. 
+All notable changes to Proksee will be documented in this file.
+
+v1.0.0a6
+
+- Modifies the Mash database download procedure to write less output.
+- Correctly instructs QUAST to use the user-specified minimum contig size.
+- Better handling of Mash for species estimation of large assemblies with many contigs.
+- Mash may now be run using parrellization.
```

Comment:
Change **parrellization** to **parallelization**

Reply:
Thanks! I can't spell! Haha.

Reply:
Heh, nor can I 😛